pkgs/development/libraries/libajantv2/default.nix

```
@@ -0,0 +1,33 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, cmake
+, ninja
+, pkg-config
+}:
+
+stdenv.mkDerivation rec {
+  pname = "libajantv2";
+  version = "16.2-bugfix5";
+
+  src = fetchFromGitHub {
+    owner = "aja-video";
+    repo = "ntv2";
+    rev = "v${version}";
+    sha256 = "sha256-h5PKWMwqTeI5/EaTWkjYojuvDU0FyMpzIjWB98UOJwc=";
+  };
+
```

I would also suggest to split the outputs here to binaries and development files. It should do that automatically in this case (for `bin` and `include`+`lib`):
```suggestion

outputs = [ "out" "dev" ];

```

---

Thanks for the suggestion. I've added it to the derivation, but right know I don't really understand it as it seems to still pack everthing (bin + lib) in one output?

```
> ls -al /nix/store/gr95i264ij66l1hh1cqzq483agsdm2yn-librist-0.2.6/
total 11146
dr-xr-xr-x    4 root root       4 Jan  1  1970 .
drwxrwxr-t 2097 root nixbld 10712 Oct  5 23:54 ..
dr-xr-xr-x    2 root root       6 Jan  1  1970 bin
dr-xr-xr-x    2 root root       5 Jan  1  1970 lib
```

Is there also more you can recommend me about this separation/using it I can read/look through as example?

---

Sorry my bad, fixed